app/js/features/prices/reducer.js

```diff
@@ -7,12 +7,18 @@ function reducer(state = {}, action) {
       ...state,
       ...action.data
     };
+  case FETCH_PRICES_FAILED:
+    return {
+      ...state,
+      ...{error: action.error}
+    };
   default:
     return state;
   }
 }
 
 export default reducer;
 
-export const getEthUsdPrice = state => state.prices.ETH.USD;
-export const getSntUsdPrice = state => state.prices.SNT.USD;
+export const getEthUsdPrice = state => state.prices.ETH && state.prices.ETH.USD;
+export const getSntUsdPrice = state => state.prices.SNT && state.prices.SNT.USD;
+export const hasPricesError = state => !!state.prices.error
```

Comment:
I would change this to `getPricesError` and in the component, you can still check for it's truthiness, but also display the actual error.